lib/rouge/lexers/bpf.rb

```diff
@@ -13,25 +13,39 @@ class BPF < RegexLexer
       ).join('|')
 
       MISC_KEYWORDS = %w(
-        be16 be32 be64 exit lock
+        be16 be32 be64 exit lock map
       ).join('|')
 
       state :root do
+        # Line numbers and hexadecimal output from bpftool/objdump
+        rule %r/(\d+)(:)(\s+)(\(\h{2}\))/i do
+          groups Generic::Lineno, Punctuation, Text::Whitespace, Generic
```

Comment:
```suggestion
          groups Generic::Lineno, Punctuation, Text::Whitespace, Literal::Number::Hex
```

Is there a reason not to use the hexadecimal token?

Reply:
I'm not sure they should be highlighted. They're not very relevant to the output. I just want to ensure their presence doesn't break highlighting of the actual bytecode (the rest of the line).

Reply:
Gotcha.